app.py

```diff
@@ -1,62 +1,71 @@
 from selenium import webdriver
 from selenium.webdriver.common.keys import Keys
-import time
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.support.ui import WebDriverWait
 
 class Twitbot:
-    def __init__(self,user,password):
-        self.user = user
-        self.password = password
-        self.bot = webdriver.Firefox()
-
-    def Login(self):
-        bot = self.bot
-        bot.get('https://twitter.com/')
-        time.sleep(5)
-        user = bot.find_element_by_class_name('email-input')
-        password = bot.find_element_by_name('session[password]')
-        user.send_keys(self.user)
-        password.send_keys(self.password)
-
-        password.send_keys(Keys.RETURN)
-        time.sleep(5)
-
-    def GetTweets(self,hashtag):
-        bot = self.bot
-        bot.get("https://twitter.com/search?q=" + hashtag + "&src=typeahead_click")
+  def __init__(self, user, password, email):
+    self.user = user
+    self.password = password
+    self.email = email
+    self.driver = webdriver.Firefox()
+
+  def Login(self):
+    bot = self.driver
+    bot.get('https://twitter.com/login')
+    bot.implicitly_wait(10) # seconds
+
+    emailInput = bot.find_element_by_class_name('js-username-field')
+    emailInput.send_keys(self.user)
+
+    passwordInput = bot.find_element_by_class_name('js-password-field')
+    passwordInput.send_keys(self.password)
+    passwordInput.send_keys(Keys.RETURN)
+
+    print(bot.current_url)
+
+# TODO: Solves the challenge page
+# def solveChallenge():
+#   dom contains 'Help us keep your account safe.'
+#   url contains login_challenge
```

Comment:
If the account used is created on the machine running the BOT there are no problems with:
"Help us keep your account safe"

This is caused by the change on OS and IP detected by Twitter.